### PR TITLE
feat: import from GeoDataFrames

### PR DESCRIPTION
**Pull request by svenruf**
Implemented import functions that warp the pd.rename function. 
Created tests accordingly.

With importing from different sources the problem mentioned in #18 becomes more relevant.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - python=3.8
+- python=3.8
```
Comment (henrymartin1) [inline environment.yml:6]: didn't we fix this already?

**Comment by henrymartin1** [comment]
Hey Sven, 
thanks a lot for the pull request it looks like a lot of cool stuff to integrate! Before we can start I have two questions:
- It looks like the branch is not up to date with the repository (we had some commits yesterday). 
- It seems that this pull request combines several new features (e.g., the tutorial?). In general it is important to have 1 pull request per new feature because otherwise it gets very hard to review. If it happens by accident it is usually no problem to integrate the two features at once or separate them into two pull requests (depending on the submission)

**Comment by svenruf** [comment]
Hi, yes I saw this too, I was not sure how to correctly merge the updates into my development branch. 

For the other features, I actually tried to seperate the features using different branches, but maybe I acidentally merged them once. May be we have a look at this tomorrow?

File: environment.yml (hunk 1 above)
Comment (svenruf) [reply environment.yml:6]: yes, seems to be the same problem with the not correct merge.

**Comment by svenruf** [comment]
closed pull request because of too many false commits in the development branch. New, cleaner pull request will follow.

**Pull request closed without merging**